### PR TITLE
Fix one qubit gate docstrings to single standard, start gate zoo

### PR DESCRIPTION
**Pull request by dabacon**
This PR updates the doc strings for single qubit gates.  These gates now all render correctly on quantumai.google.  

* This moves over to using LaTeX for the documentation. In some cases this makes the code doc string less readable, in some case it makes it is more readable.  It renders nicely on quantumai.google

This also starts #852 for single qubit gates. Adding two and higher qubit gates will come in a follow up PR assuming the pattern in this PR is good.

### CODE REVIEW

File: docs/gatezoo.ipynb [hunk 1]
```diff
@@ -0,0 +1,996 @@
...
+    "    <a target=\"_blank\" href=\"https://quantumai.google/cirq/qubits\"><img src=\"https://quantumai.google/site-assets/images/buttons/quantumai_logo_1x.png\" />View on QuantumAI</a>\n",
+    "  </td>\n",
+    "  <td>\n",
+    "    <a target=\"_blank\" href=\"https://colab.research.google.com/github/quantumlib/Cirq/blob/master/docs/qubits.ipynb\"><img src=\"https://quantumai.google/site-assets/images/buttons/colab_logo_1x.png\" />Run in Google Colab</a>\n",
+    "  </td>\n",
+    "  <td>\n",
+    "    <a target=\"_blank\" href=\"https://github.com/quantumlib/Cirq/blob/master/docs/qubits.ipynb\"><img src=\"https://quantumai.google/site-assets/images/buttons/github_logo_1x.png\" />View source on GitHub</a>\n",
+    "  </td>\n",
+    "  <td>\n",
+    "    <a href=\"https://storage.googleapis.com/tensorflow_docs/Cirq/docs/qubits.ipynb\"><img src=\"https://quantumai.google/site-assets/images/buttons/download_icon_1x.png\" />Download notebook</a>\n",
```
Comment (MichaelBroughton) [inline docs/gatezoo.ipynb:55]: Nit: Links out to wrong thing.

File: docs/qudits.ipynb [hunk 2]
```diff
@@ -11,7 +11,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 1,
```
Comment (MichaelBroughton) [inline docs/qudits.ipynb:14]: Why changes here ?

**Review by MichaelBroughton — APPROVED** [review]
LGTM after nits.

File: docs/gatezoo.ipynb (hunk 1 above)
Comment (dabacon) [reply docs/gatezoo.ipynb:55]: fixed

**Pull request merged**